pkg/kudoctl/cmd/env/kudohome/home.go

```diff
@@ -0,0 +1,31 @@
+package kudohome
+
+import (
+	"os"
+	"path/filepath"
+)
+
+// Home describes the location of a CLI configuration.
+type Home string
+
+// String returns Home as a string with expansions $HOME == home
+func (h Home) String() string {
+	return os.ExpandEnv(string(h))
+}
+
+// Path returns Home with elements appended.
+func (h Home) Path(elem ...string) string {
```

Comment:
what are elements? 🤔 is that like parts of the file path?

Reply:
It is a convince function for "things" and paths relative to home.
that said it is only used internal to `Home` so I'll make it private.